template/webpack.config.js

```diff
@@ -5,24 +5,26 @@ const WasmPackPlugin = require("@wasm-tool/wasm-pack-plugin");
 const dist = path.resolve(__dirname, "dist");
 
 module.exports = {
-  mode: "production",
+  mode: "development",
+  experiments: {
+    futureDefaults: true,
```

Comment:
I assume this is being set as Webpack 5 is sort of in a weird spot where it doesn't choose sync wasm (like in Webpack 4) or async wasm (intended to be default in future webpack versions).  If this is the case and this line is meant to just set `experiments.asyncWebAssembly` to `true`, it might be simpler to only set that or at the very least to leave a comment explaining that that is why `futureDefaults` is being set.

Reply:
I believe you're right, will push the suggested change.